frontend/public/components/overview/resource-overview-page.tsx

```diff
@@ -10,17 +10,22 @@ import {
 } from '../utils';
 
 import { BuildOverview } from './build-overview';
+import { KnativeOverview } from './knative-overview';
 import { NetworkingOverview } from './networking-overview';
 import { OverviewItem } from '.';
 import { resourceOverviewPages } from './resource-overview-pages';
 
 const { common } = Kebab.factory;
 const menuActions = [...common];
 
-export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps>= ({item: {buildConfigs, routes, services}}) => (
+export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps>= ({item: {buildConfigs, routes, services, ksroutes, revisions, configurations}}) => (
   <div className="overview__sidebar-pane-body">
-    <BuildOverview buildConfigs={buildConfigs} />
-    <NetworkingOverview services={services} routes={routes} />
+    {!configurations ? (
+      <React.Fragment>
+        <BuildOverview buildConfigs={buildConfigs} />
+        <NetworkingOverview services={services} routes={routes} />
+      </React.Fragment>
+    ) : <KnativeOverview ksroutes={ksroutes} configurations={configurations} revisions={revisions} />}
```

Comment:
We need to find a way to plug into the console without adding knative specific branching logic.